oak/proto/introspection_events.proto

```diff
@@ -47,15 +47,11 @@ message NodeDestroyed {
 }
 
 message ChannelCreated {
-  uint64 node_id = 1;
-
-  uint64 channel_id = 2;
+  uint64 channel_id = 1;
 }
 
 message ChannelDestroyed {
-  uint64 node_id = 1;
-
-  uint64 channel_id = 2;
+  uint64 channel_id = 1;
 }
```

Comment:
It did appear to me that the idiomatic proto thing to do would be to keep the original field number of two. However since this proto is only consumed by ourselves it seemed nice to keep the one field of the message as field number 1.